conf/container.config

```diff
@@ -0,0 +1,20 @@
+process {
+    withLabel: simulation_image {
+        container = "olivierlabayle/targene-simulations:0.0"
+    }
+    withLabel: targenecore_image {
+        container = "olivierlabayle/tl-core:summary_output"
+    }
+    withLabel: ukb_image {
+        container = "olivierlabayle/ukbmain:sha-412c12e"
+    }
+    withLabel: tmle_image {
+        container = "olivierlabayle/targeted-estimation:sha-507f32f"
```

Comment:
Same with these, should these be new version releases? Or what is the motivation behind this naming system?